_minutes/2022-01-20-wecg.md

```diff
@@ -0,0 +1,124 @@
+Call-in details: [WebExtensions CG, 20th January 2022](https://www.w3.org/events/meetings/d7bbce8f-549f-46ea-b440-ea6902f8707c/20220120T080000)
+Zoom issues? Ping @zombie (Tomislav Jovanovic) in [chat](https://github.com/w3c/webextensions/blob/main/CONTRIBUTING.md#joining-chat)
+
+
+## Agenda: [discussion in #144](https://github.com/w3c/webextensions/issues/144), [other issues](https://github.com/w3c/webextensions/issues)
+
+The meeting will start at 3 minutes after the hour.
+
+ * **Carry-over from previous meetings**
+   * (nothing)
+ * **Other new issues**
+   * #145 Proposals for improving WECG meetings (https://github.com/w3c/webextensions/issues/145)
+   * #146 Clarify browser vendor positions on APIs in MV3 (https://github.com/w3c/webextensions/issues/146)
+ * **Open discussion queue (add yourself at the bottom)**
+   * [Ales Pospichal] Manifest V3 vs native messaging - final decision / next steps on service worker limit (https://github.com/w3c/webextensions/issues/120)
+   * [Tomislav]  #19 request: APIs and infrastructure to simplify cross-browser automated testing of extensions (https://github.com/w3c/webextensions/issues/19#issuecomment-1011492948)
+     * Mozilla's web-ext https://github.com/mozilla/web-ext
+     * An existing use case, in lib and CLI form, “cross-browser” (currently uses internal/debugging protocols to do stuff).
+   * #115 move browser specific settings to browser_specific_settings key  (Carlos) (https://github.com/w3c/webextensions/issues/115)
+   * #147 Inconsistency: determine when non-persistent background pages get suspended (https://github.com/w3c/webextensions/issues/147)
+
+
+## Attendees (sign yourself in)
+
+ 1. Ales Pospichal (I.CA)
+ 2. Alexei (Privacy Badger)
+ 3. Rob Wu (Mozilla)
+ 4. Giorgio Maone (NoScript)
+ 5. Nick McGuire (1Password)
+ 6. Carlos Jeurissen (Jeurissen Apps)
+ 7. Philipp Claßen (Ghostery)
+ 8. Simeon Vincent (Google)
+ 9. Oliver Dunk (1Password)
+ 10. Krzysztof Modras (Ghostery)
+ 11. Mukul Purohit (Microsoft)
+ 12. Richard Worth (Capital One)
+ 13. Emilia Paz (Google)
+ 14. Bradley Cushing (Dashlane)
+ 15. Nir Nahum (WalkMe)
+ 16. Jessie Berlin (Apple)
+ 17. Tomislav Jovanovic (Mozilla)
+ 18. Mélanie Chauvel (Dashlane)
+
+
+## Meeting notes
+
+[Issue 145](https://github.com/w3c/webextensions/issues/145): Proposals for improving WECG meetings
+
+ * [simeon] Introduced the topic in the last meeting, and elaborated in issue 145. Not that much to discuss today; encourage everyone to take a look and offer feedback.
+ * [tomislav] Timeboxing ([issue 144](https://github.com/w3c/webextensions/issues/144)) is a way to run a meeting. That suggestion isn't very controversial.
+ * [simeon] Idea of “champions” is just thinking aloud loud. Encourage everyone to take a look at the “Keeping meetings productive” section.
+ * [tomislav] Discussions that can be asynchronous/offline (i.e. in issues on Github) can better be discussed asynchronously.
```

Comment:
```suggestion
 * [tomislav] Discussions that can be asynchronous/offline (i.e. in issues on Github) can better be discussed asynchronously.
 * [jessie] The "champions" suggestion can be very effective for driving a topic or project to conclusion.
```

I'm not sure if Jessie has a GitHub account, so I'm following up over email to make sure she's comfortable with this suggestion.